install_deps.sh

```diff
@@ -300,36 +300,20 @@ install_openssl() {
 # https://github.com/grpc/grpc/blob/master/test/distrib/cpp/run_distrib_test_cmake.sh
 install_grpc() {
     cd ${HOME}
-    git clone -b v1.37.1 --depth 1 --recurse-submodules https://github.com/grpc/grpc && \
-        cd grpc && \
-        mkdir -p ${HOME}/grpc/third_party/abseil-cpp/cmake/build && \
-        cd ${HOME}/grpc/third_party/abseil-cpp/cmake/build && \
-        cmake -DCMAKE_BUILD_TYPE=Release \
-            -DCMAKE_POSITION_INDEPENDENT_CODE=TRUE \
-            -DCMAKE_INSTALL_PREFIX=/usr/local \
-            ../.. && \
-        make -j$(nproc) install && \
-        mkdir -p ${HOME}/grpc/third_party/protobuf/cmake/build && \
-        cd ${HOME}/grpc/third_party/protobuf/cmake/build && \
-        cmake -DBUILD_SHARED_LIBS=ON \
-            -DCMAKE_BUILD_TYPE=Release \
-            -DCMAKE_INSTALL_PREFIX=/usr/local \
-            .. && \
-        make -j$(nproc) install && \
-        mkdir -p ${HOME}/grpc/cmake/build && \
-        cd ${HOME}/grpc/cmake/build && \
-        cmake -DgRPC_INSTALL=ON \
-            -DgRPC_ABSL_PROVIDER=package \
-            -DgRPC_PROTOBUF_PROVIDER=package \
-            -DgRPC_SSL_PROVIDER=package \
-            -DBUILD_SHARED_LIBS=ON \
-            -DCMAKE_BUILD_TYPE=Release \
-            -DCMAKE_INSTALL_PREFIX=/usr/local \
-            ../.. && \
-        make -j$(nproc) install &&
-        cd ${HOME} && \
-        rm -r ${HOME}/grpc
-
+git clone -b v1.48.x --depth 1 --recurse-submodules https://github.com/grpc/grpc && \
```

Comment:
backslashes are redundant here, since it's a function

Reply:
@toly-kournik all the functions in this file use backslashes, I believe this is for readability.

Reply:
It less readable and it was just copied as is when converted to functions :-(

Reply:
@toly-kournik I have updated the install_deps file to remove redundant backslashes.